src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Platform/Mono/MonoPlatform.ts

```diff
@@ -26,22 +29,31 @@ export const monoPlatform: Platform = {
   },
 
   findMethod: function findMethod(assemblyName: string, namespace: string, className: string, methodName: string): MethodHandle {
-    // TODO: Cache the assembly_load outputs?
-    const assemblyHandle = assembly_load(assemblyName);
+
+    var assemblyHandle: number | undefined = registeredAssemblies[assemblyName];
     if (!assemblyHandle) {
-      throw new Error(`Could not find assembly "${assemblyName}"`);
+      assemblyHandle = assembly_load(assemblyName);
+      if (!assemblyHandle) {
+        throw new Error(`Could not find assembly "${assemblyName}"`);
+      }
+      registeredAssemblies[assemblyName] = assemblyHandle;
     }
-
-    const typeHandle = find_class(assemblyHandle, namespace, className);
+    var typeHandle: number | undefined = registeredClasses["[${assemblyName}]${namespace}.${className}"];
```

Comment:
This is broken because it uses the wrong quote style. Should be backtick, not double-quote, so as to enable string interpolation.

Reply:
Also below.

Reply:
oops my bag; noob error